src/test/scala/org/neo4j/spark/Neo4jDataFrameScalaTest.scala

```diff
@@ -62,17 +62,18 @@ class Neo4jDataFrameScalaTest {
   }
 
   @Test def mergeEdgeListWithRelProperties {
-    val rows = sc.makeRDD(Seq(Row("Laurence", "Keanu", "Mentor")))
+    val rows = sc.makeRDD(Seq(Row("Laurence", "Keanu", "Mentor", Seq("1980"))))
     val schema = StructType(Seq(
       StructField("src_name", DataTypes.StringType),
       StructField("dst_name", DataTypes.StringType),
-      StructField("screen", DataTypes.StringType)
+      StructField("screen", DataTypes.StringType),
+      StructField("met", DataTypes.createArrayType(DataTypes.StringType, false))
     ))
     val df = new SQLContext(sc).createDataFrame(rows, schema)
     val rename = Map("src_name" -> "name", "dst_name" -> "name")
-    Neo4jDataFrame.mergeEdgeList(sc, df, ("Person",Seq("src_name")),("ACTED_WITH",Seq("screen")),("Person",Seq("dst_name")),rename)
```

Comment:
Is this working only for relationships? Cay you apply it also for nodes?

Reply:
It can also be applied for nodes. I've modified the test to reflect this.

Reply:
Can you please apply the changes also to the `Neo4jDataFrame.createNodes` and add a test for it?

Reply:
It's already been applied for `createNodes`.
Just added a test for it in the most recent commit.